src/etc/inc/openvpn.inc

```diff
@@ -1096,8 +1106,32 @@ function openvpn_reconfigure($mode, $settings) {
 					$tls_directive = "tls-auth";
 					if ($mode == "server") {
 						$tlsopt = 0;
+						//switch($settings['tlsauth_keydir']){
```

Comment:
Why add all of this commented out? The setting should also be valid for servers, provided that it is set properly compared to the client (one side set to "1", the other "0", or both set to omit). It would need code on the server page, of course.